tasks.py

```diff
@@ -666,9 +666,10 @@ def _replace(pattern):
 @task
 def test(ctx):
     """Run unit tests."""
-    run("go test -short ./...")
-    run("go test -short -race ./...")
-
+    envtest_asset_dir = os.getcwd() + "/dev-env/unittest"
+    run("source {}/setup-envtest.sh; fetch_envtest_tools {}; setup_envtest_env {}; go test -short ./...".format(envtest_asset_dir, envtest_asset_dir, envtest_asset_dir), echo=True)
+    run("source {}/setup-envtest.sh; fetch_envtest_tools {}; setup_envtest_env {}; go test -short -race ./...".format(envtest_asset_dir, envtest_asset_dir, envtest_asset_dir), echo=True)
+    run("sudo rm -rf {}/bin".format(envtest_asset_dir), echo=True)
```

Comment:
I'd just add it to gitignore and not remove it, otherwise every time we run unit tests (locally) we'll download everything again.